src/katgpucbf/xbgpu/main.py

```diff
@@ -51,56 +51,119 @@
 from .correlation import device_filter
 from .output import BOutput, XOutput
 
+_OD = TypeVar("_OD", bound="OutputDict")
 logger = logging.getLogger(__name__)
+DEFAULT_HEAP_ACCUMULATION_THRESHOLD = 52
```

Comment:
I'm tempted to say let's just take the opportunity to make it required for the user to pass it, rather than making an arbitrary default that's only applicable for one use case. But I also don't want to hold up this PR for longer, so maybe it should be a ticket.